hack/mirror-origin-tests.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+. $(dirname "$0")/origin-tests-common.sh
+
+skopeo_workdir=$(mktemp -d)
+mapping_file=$(mktemp)
+function finish {
+  rm -rf ${skopeo_workdir}
+  rm -f ${mapping_file}
+}
+trap finish EXIT
+
+MIRROR_ORIGIN_TESTS_OC_TOOL="${MIRROR_ORIGIN_TESTS_OC_TOOL:-oc}"
```

Comment:
wait actually, I think this is the first time we ever introduce a dependency on OC into cnf-features-deploy
do we want that?
I understand this is just in order to do the actual mirroring

maybe we should add a test if it doesnt exist
or we can even help people get it from:
https://github.com/openshift/okd/releases/latest
I'd at least suggest that in the error msg if it doesnt exists

unfortunately RH doesnt make it easy enough to automate the d/l of the latest oc

Reply:
we are using oc in the setup-test-cluster.sh script also:
https://github.com/openshift-kni/cnf-features-deploy/blob/5f35374d8a50fc3db838ac5f55a9f612698a7607/hack/setup-test-cluster.sh#L7